deploy/swagger-template.yaml

```diff
@@ -406,6 +408,131 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /orphanvolumes:	
```

Comment:
Add the latest modifications that introduce `successfulOrphanDeletions` and `failedOrphanDeletions`, and generate the client-sdk again.